CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # 2.2.1 Release notes
 
+### Bugfixes
```

Comment:
This will merge to the wrong section now that we have released again. Don't forget to update to the latest master and merge these notes to the next release section.